src/MeetingsSDKAdapter.test.js

```diff
@@ -486,108 +486,105 @@ describe('Meetings SDK Adapter', () => {
     });
   });
 
-  describe('audioControl()', () => {
-    test('returns the display data of a meeting control in a proper shape', (done) => {
-      meetingsSDKAdapter.audioControl(meetingID).subscribe((dataDisplay) => {
-        expect(dataDisplay).toMatchObject({
-          ID: 'mute-audio',
-          icon: 'microphone-muted_28',
-          tooltip: 'No microphone available',
-          state: 'disabled',
-          text: null,
-        });
-        done();
-      });
-    });
-
-    test('throws errors if sdk meeting object is not defined', (done) => {
-      meetingsSDKAdapter.audioControl('inexistent').subscribe(
-        () => {},
-        (error) => {
-          expect(error.message).toBe('Could not find meeting with ID "inexistent"');
-          done();
-        },
-      );
-    });
-  });
-
-  describe('handleLocalAudio()', () => {
+  describe('handleLocalAudio() when local audio track is unmuted', () => {
     beforeEach(() => {
       meetingsSDKAdapter.meetings[meetingID] = {
         ...meeting,
         localAudio: {
           stream: mockSDKMediaStreams.localAudio,
+          permission: undefined,
         },
+        disabledLocalAudio: null,
         remoteAudio: {},
       };
     });
 
-    test('skips muting audio if there is an inactive meeting', async () => {
+    test('does not call muteAudio function of Sdk if there is an inactive meeting', async () => {
       meetingsSDKAdapter.meetings[meetingID].remoteAudio = null;
       await meetingsSDKAdapter.handleLocalAudio(meetingID);
 
       expect(mockSDKMeeting.muteAudio).not.toHaveBeenCalled();
     });
 
-    test('skips unmuting audio if there is an inactive meeting', async () => {
-      meetingsSDKAdapter.meetings[meetingID].remoteAudio = null;
+    test('mutes audio if the the audio track is unmuted', async () => {
       await meetingsSDKAdapter.handleLocalAudio(meetingID);
 
-      expect(mockSDKMeeting.unmuteAudio).not.toHaveBeenCalled();
+      expect(mockSDKMeeting.muteAudio).toHaveBeenCalled();
     });
 
-    test('mutes audio if the the audio track is enabled', async () => {
+    test('sets localAudio.stream property to null once the audio track is muted', async () => {
```

Comment:
sets localAudio.stream to null and disabledLocalAudio to the audio stream

or just: mutes audio